pkg/util/etcdutil/migrate.go

```diff
@@ -18,6 +19,8 @@ import (
 const (
 	apiserverAddr = "http://127.0.0.1:8080"
 	etcdServiceIP = "10.3.0.15"
+
+	etcdClusterName = "etcd-cluster"
```

Comment:
system-etcd or kube-etcd both seem good.

Then you would interact with it as:

```
kubectl get etcdclusters/kube-etcd
or
kubectl get etcdclusters/system-etcd
```

@yifan-gu any bikeshed thoughts here?